metrics/prepare.sh

```diff
@@ -19,9 +19,9 @@ flutterCreate() {
 flutterCreate 'perf-test-app-plain'
 flutterCreate 'perf-test-app-with-sentry'
 
-# bump minSdkVersion to 19
+# bump minSdkVersion to 21
 gradleFile="$targetDir/perf-test-app-with-sentry/android/app/build.gradle"
-sed "s/flutter.minSdkVersion/19/g" "$gradleFile" > "$gradleFile.new" && mv "$gradleFile.new" "$gradleFile"
+sed "s/flutter.minSdkVersion/21/g" "$gradleFile" > "$gradleFile.new" && mv "$gradleFile.new" "$gradleFile"
```

Comment:
we can revert this as well right?

Reply:
Does not build with version 19, it seems most recent version of sentry-android does not support 19?

Reply:
<img width="1281" alt="Bildschirmfoto 2025-02-06 um 11 10 52" src="https://github.com/user-attachments/assets/0daeb7cb-05c2-482f-ad5f-61f0cc2406f8" />

Reply:
ah yeah android bumped api from 19 to 21, let's leave it then. but we can do the minSDK bump separately